AzureMS/scripts/npc/9062000.js

```diff
@@ -232,39 +232,39 @@ function SecondJob(i) {
     switch (i) {
         case 100:
             v2 += "#L112##bHero����  " + r + "Fighter �� Crusader �� Hero" + s + "";
-            v2 += "#L122##bPaladin����  " + r + "Page �� Night �� Paladin" + s + "";
+            v2 += "#L122##bPaladin����  " + r + "Page �� White Knight �� Paladin" + s + "";
             v2 += "#L132##bDark Knight  " + r + "Spearman �� Berserker �� Dark Knight" + s + "";
             break;
 
         case 200:
-            v2 += "#L212##bArchmage (Fire, Poison)  " + r + "Wizard �� Mage �� Archmage" + s + "";
-            v2 += "#L222##bArc Mage (Sun, Cole)  " + r + "Wizard �� Mage �� Archmage" + s + "";
+            v2 += "#L212##bArchMage (Fire, Poison)  " + r + "Wizard �� Mage �� Archmage" + s + "";
+            v2 += "#L222##bArchMage (Ice, Lightning) " + r + "Wizard �� Mage �� Archmage" + s + "";
             v2 += "#L232##bBishop#e����#n����������" + r + "Cleric �� Priest �� Bishop" + s + "";
             break;
 
         case 300:
             v2 += "#L312##bBowmaster  " + r + "Hunter �� Ranger �� Bowmaster" + s + "";
-            v2 += "#L322##bShrine������  " + r + "Shooter �� Sniper �� Shrine" + s + "";
+            v2 += "#L322##bMarksman�� " + r + "Crossbowman �� Sniper �� Marksman" + s + "";
             break;
 
         case 301:
-            v2 += "#L332##bPathfinder  " + r + "Ancient Archer �� Chaser �� Pathfinder" + s + "";
+            v2 += "#L332##bPathfinder  " + r + "Ancient Archer �� Soulchaser �� Pathfinder" + s + "";
 			break;
        
         case 400:
-            v2 += "#L412##bNight Lord��  " + r + "Assassin �� Hermit �� Night Lord" + s + "";
-            v2 += "#L422##bShadow������  " + r + "Sheep �� Sheepmaster �� Shadow" + s + "";
-            v2 += "#L434##bDual Blader  " + r + "Dewarer �� Slasher �� Dual Blader" + s + "";
+            v2 += "#L412##bNight Lord�� " + r + "Assassin �� Hermit �� Night Lord" + s + "";
+            v2 += "#L422##bShadower���� " + r + "Bandit �� Chief Bandit �� Shadower" + s + "";
+            v2 += "#L434##bDual Blade  " + r + "Blade Recruit �� Blade Acolyte �� Blade Master" + s + "";
```

Comment:
Missing `Blade Specialist` and `Blade Lord`.
Job | Level for Job Advancement
:---|:---
Rogue | 10
Blade Recruit | 20
Blade Acolyte | 30
Blade Specialist | 45
Blade Lord | 60
Blade Master | 100